### DISCUSSION TITLE
why exclude project folders when importing .zip?

### DISCUSSION THREAD

**Discussion by aekobear** [godotengine/godot-proposals#6196] [Editor]
When trying to import a .zip from my own computer, all my Godot project folders are removed from the file picker. I couldn't find any information on this, but I can only assume it was an intentional decision.

I'm curious is this is valuable to people? From my POV it's frustrating because my workflow is generally:

1) Make a new Godot project to build out a reusable game mechanic
2) zip up the folder containing the useful stuff (ignoring all the test/debug scenes and files)
3) Import the zip into my main project

Because of the restriction, I'm forced to move the zips out into a different directory outside of the project. But I feel like for someone who *wants* to have all their zips in a separate directory, they could just opt in to that structure and wouldn't need a filter in the file picker to help them find their zip folder

![image](https://user-images.githubusercontent.com/25251056/215291248-c405554e-f859-4169-ab5b-0b6d52032903.png)

**Comment by timothyqiu** [comment]
You can get access to those folders by clicking the eye icon in the top row.
